app/script/media/MediaEmbeds.js

```diff
@@ -89,10 +89,12 @@ z.media.MediaEmbeds = (function() {
    * @returns {string} Youtube embed URL
   */
   const _generate_youtube_embed_url = function(url) {
-    if (url.match(/youtu.be|youtube.com/)) {
+    if (url.match(_regex.youtube)) {
 
       const video_id = url.match(/(?:embed\/|v=|v\/|be\/)([a-zA-Z0-9_-]{11})/);
-      if (!video_id) return;
+      if (!video_id) {
+          return;
```

Comment:
I tried to use https://marketplace.visualstudio.com/items?itemName=lonefy.vscode-JS-CSS-HTML-formatter unfortunately it started to reindent everything so I disabled it, I'll use editorconfig then, thanks.